src/inverseRobot.js

```diff
@@ -25,6 +25,17 @@
  */
 function inverseRobot(robot) {
   // write code here
+  const newRobot = {};
+
+  for (const key in robot) {
+    if (Object.keys(newRobot).includes(robot[key])) {
+      return null;
+    } else {
+      newRobot[robot[key]] = key;
+    }
```

Comment:
```suggestion
    }
    newRobot[robot[key]] = key;
```
Можно else не писать, если в if происходит return